### PR TITLE
Rewrite the event routes so that they work as the blog ones

### LINKED ISSUE #48: events aren't being listed at the moment
Presumably this is because none them have their event date set - we should go through and manually set the data on at least the most recent 5 events on prose.io for the purposes of demoing the site.

This refers to both the front page and the events page:

* http://sinatra.staging.shineyoureye.org/
* http://sinatra.staging.shineyoureye.org/info/events

### PR DESCRIPTION
**Pull request by octopusinvitro**
In order to serve the site, we run the Sinatra app and then wget it
and push it to a gh-pages branch, so that it is served as a static site.

The event list page at `/info/events` works when running the app
locally, but it throws a 404 once we wget it and serve it as static.

This is because the route /info/events (the event list page) is scraped
first by wget, creating an HTML file info/events.html. Later the route
/info/events/EVENT_SLUG is scraped, and wget replaces info/events.html
with an info/events directory containing a EVENT_SLUG.html file.

In other words, the index page is saved and then deleted by wget.

To solve this, we are making the URL path for an individual event
look like /events/EVENT_SLUG instead of /info/events/EVENT_SLUG.

## Related issues

Closes #48

### CODE REVIEW

File: tests/web/events.rb [hunk 1]
```diff
@@ -33,4 +33,12 @@
       refute_empty(single_event.css('div'))
     end
   end
+
+  describe 'redirection from old route' do
+    before { get '/info/events' }
+
+    it 'redirects to events route' do
+      subject.css('h1').first.text.must_equal('Events')
```
Comment (mhl) [inline tests/web/events.rb]: This won't work, I'm afraid. `/info/events` is just returning a static HTML page with a refresh tag in it, nothing cleverer than that. In this test you should probably find that `<meta>` tag and check that its `content` value is as expected.

One other thing is that you'll have to add `/info/events` to the end of `/scraper-start-page.html` to make sure it's actually crawled.
